pkg/scaffold/files/project.go

```diff
@@ -14,21 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package project
+package files
```

Comment:
> pkg/scaffold/v1 -> pkg/scaffold/files/v1
> pkg/scaffold/v2 -> pkg/scaffold/files/v2

Why we should add more one dir here? What is the advantage which justifies this change?

Reply:
Kubebuilder has three kind of objects. In the first level we have the Cobra commands. The second layer is the scaffolders. The last one is the files. Using `files` as a directory allows the directory structure to represent the conceptual design of kubebuilder.

Reply:
I'd suggest you do a specific PR with only the layout changes and make clear the motivations for your suggestion of changes. In fact, IHMO this kind of suggestion/change would be better addressed by raising an issue with and then do a PR only/if the maintainers accept and agree with.